specification/marketplacecatalog/resource-manager/Products/preview/2022-09-01-preview/products.json

```diff
@@ -0,0 +1,1411 @@
+{
+  "x-generator": "NSwag v13.12.1.0 (NJsonSchema v10.4.6.0 (Newtonsoft.Json v13.0.0.0))",
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Marketplace Catalog Browse API",
+    "description": "Microsoft Marketplace Catalog Browse API",
+    "version": "2022-09-01-preview"
+  },
+  "host": "catalogapi.azure.com",
```

Comment:
Is this host name correct?

Reply:
Yes this host name is correct

Reply:
Have changed the host name to "management.azure.com"